Sources/SystemMetrics/SystemMetrics.swift

```diff
@@ -363,6 +370,7 @@ public enum SystemMetrics {
 
     private static let cpuUsageCalculator = CPUUsageCalculator()
 
+    @Sendable
```

Comment:
Why are those needed? `@Sendable` on methods is normally not required to be stated explicitly